java/vector/src/main/java/org/apache/arrow/util/AutoCloseables.java

```diff
@@ -1,11 +1,13 @@
-/*
- * Portions Copyright (C) 2017-2018 Dremio Corporation
```

Comment:
@jacques-n this conflicted with the style check. Here I moved it down to a separate comment to get the check to pass, but let me know how you would like to format it.  Alternatively, I think we could define a `RegexHeader` rule that allows for an optional first line.

Reply:
on second thought, it's probably easiest to just suppress the check for these 2 files and keep them as they were